docs/en/styling-and-theming/basic-usage.md

```diff
@@ -0,0 +1,177 @@
+# Basic Usage
```

Comment:
@agubler referencing [your comment on the main PR](https://github.com/dojo/framework/pull/300#issuecomment-477884342) - do you see some of these specifically as not providing enough detail in their opening sentences? I agree there should be some context around the examples, but I also see the sample code itself as (hopefully) self-descriptive enough to make the usage fully understandable, together with the introductory sentence. The links to the supplemental guide were only really intended for quick navigation to specific areas of interest if a user wants to go into full detail for a feature - not so that they need to read up on something before understanding the sample usage. Happy to add more context to some of these if they're not clear enough as-is - and perhaps we need a better way of linking to supplemental for the above purpose? Maybe a 'See here for more detail' link at the end of the sample code?

Reply:
Added a note to clarify the ordering